deploy/csi-azurelustre-node.yaml

```diff
@@ -157,55 +142,23 @@ spec:
               cpu: 10m
               memory: 20Mi
       volumes:
+        - hostPath:
+            path: /var/lib/csi-lustre-data/
+            type: DirectoryOrCreate
+          name: csi-data-dir
         - hostPath:
             path: /var/lib/kubelet/plugins/azurelustre.csi.azure.com
             type: DirectoryOrCreate
           name: socket-dir
         - hostPath:
             path: /var/lib/kubelet/plugins_registry/
-            type: DirectoryOrCreate
+            type: Directory
           name: registration-dir
         - hostPath:
-            path: /etc/kubernetes/
-            type: DirectoryOrCreate
-          name: azure-cred
-        - hostPath:
-            path: /mnt
-            type: DirectoryOrCreate
-          name: azurelustre-cache
+            path: /var/lib/kubelet/pods/
+            type: Directory
+          name: mountpoint-dir
         - hostPath:
             path: /dev
             type: Directory
           name: host-dev
-        - hostPath:
-            path: /var
-            type: Directory
-          name: host-var
-        - hostPath:
-            path: /usr
-            type: Directory
-          name: host-usr
-        - hostPath:
-            path: /etc
-            type: Directory
-          name: host-etc
-        - hostPath:
-            path: /run/udev
-            type: Directory
-          name: host-run-udev
-        - hostPath:
-            path: /lib
-            type: Directory
-          name: host-lib
-        - hostPath:
-            path: /lib64
-            type: Directory
-          name: host-lib64
-        - hostPath:
```

Comment:
I believe the upcall for configuration changes from the kernel to userspace requires the binaries to be accessible in the host filesystem.  This manifests as a negative 10x perf impact in IOPS tests when it doesn't work.

Reply:
Note to myself: The values in question are the following:
```bash
# lctl get_param -R *.*.max_rpcs_in_flight
mdc.lustrefs-MDT0000-mdc-ffff88e8c16cc800.max_rpcs_in_flight=128
osc.lustrefs-OST0000-osc-ffff88e8c16cc800.max_rpcs_in_flight=32
```
By default they are `8`. If we want to do this approach, we need to make sure the driver is correctly configuring and using those values. Also check on any other non-default values that were configured on the nodes to ensure they're being applied correctly as well.